simtools/simtel/simtel_light_emission.py

```diff
@@ -0,0 +1,552 @@
+
+        command += " -DNUM_TELESCOPES=1"
+        command += " -I../cfg/CTA"
+        command += "iobuf_maximum=1000000000"
+        command += super()._config_option(
+            "altitude", self._site_model.get_parameter_value("corsika_observation_level")
+        )
+        # command += super()._config_option("maximum_telescopes", "1")
+        # command += super()._config_option("trigger_telescopes", "1")
+        command += super()._config_option(
+            "atmospheric_transmission",
+            self._telescope_model.get_parameter_value("atmospheric_transmission"),
+        )
+        # command += super()._config_option("trigger_current_limit", "1e10")
+        # command += super()._config_option("show", "all") # for debugging
+        # command += super()._config_option("random_state", "none")
+        # command += super()._config_option("ONLY_TRIGGERED_TELESCOPES", "0")
+        # command += super()._config_option("ONLY_TRIGGERED_ARRAYS", "0")
+        command += super()._config_option("TRIGGER_CURRENT_LIMIT", "20")
+        command += super()._config_option("TRIGGER_TELESCOPES", "1")
+        # command += super()._config_option("ARRAY_TRIGGER", "NONE")
+        command += super()._config_option("TELTRIG_MIN_SIGSUM", "7.8")
+
+        command += super()._config_option("PULSE_ANALYSIS", "-30")
+        # command += super()._config_option("SUM_BEFORE_PEAK", "-30")
+        # command += super()._config_option("ARRAY_WINDOW", "1000")
+
+        if "real" in self.default_le_config["x_pos"]:
+            _, angles = self.calibration_pointing_direction()
+            angle_theta = 180 - angles[0]
+            angle_phi = 180 - angles[1]
+            command += super()._config_option("telescope_theta", f"{angle_theta}")
+            command += super()._config_option("telescope_phi", f"{angle_phi}")
+        else:
+            command += super()._config_option("telescope_theta", 0)
+            command += super()._config_option("telescope_phi", 0)
+
+        command += super()._config_option("power_law", "2.68")
+        command += super()._config_option(
+            "input_file", f"{self.output_directory}/{self.le_application[0]}.iact.gz"
+        )
+        command += super()._config_option(
+            "output_file",
+            f"{self.output_directory}/"
+            f"{self.le_application[0]}_{self.le_application[1]}.simtel.gz\n",
+        )
+
+        return command
+
+    def _create_postscript(self, **kwargs):  # pylint: disable=unused-argument
+        """
+        writes out post-script file using read_cta
+        """
+        postscript_dir = self.output_directory.joinpath("postscripts")
+        postscript_dir.mkdir(parents=True, exist_ok=True)
+
+        command = str(self._simtel_source_path.joinpath("hessioxxx/bin/read_cta"))
+        command += " --min-tel 1 --min-trg-tel 1"
+        command += " -q --integration-scheme 4 --integration-window 7,3 -r 5"
```

Comment:
The integration parameters are the same for all telescopes? Remember we have different readout speeds.

Reply:
I have added the command line parameters.

Reply:
Ok I can also add these to the command line.